src/ListenerCollection.php

```diff
@@ -30,18 +29,16 @@ public function addListener(string $eventName, callable $listener): void
         $this->listeners[$eventName] = [$listener];
     }
 
-    public function get($id): iterable
+    public function get($eventName): iterable
     {
-        if ($this->has($id)) {
-            return $this->listeners[$id];
+        if ($this->has($eventName)) {
+            yield from $this->listeners[$eventName];
         }
-
-        throw new InvalidArgumentException('Invalid Event identifier given.');
     }
 
-    public function has($id): bool
+    public function has($eventName): bool
```

Comment:
```suggestion
    public function has(string $eventName): bool
```

Reply:
At first, I could not apply the change since the `ListenerCollection` implements the `Psr\ContainerInterface`.

Anyway, I agree that for this case, both the input and output parameters should be stricter, so I have added the `get` and `has` methods to the `ListenerLocatorInterface`.

I also believe that it was not the best use case to use the `Psr\ContainerInterface`, as it could lead to confusion.

Thank you very much for your time, I'm delighted to have you collaborating on the project. :smile: